### PR TITLE
fix: cleaning warning messages

### PR DESCRIPTION
**Pull request by Gusinuhe**
### Description

Cleaning of warning messages 

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: Tests/Builders/InteractionTrainingBuilderTests.cs [hunk 1]
```diff
@@ -25,9 +25,11 @@ protected override void InternalSetLocked(bool lockState)
             {
                 throw new NotImplementedException();
             }
-
+#pragma warning disable CS0067 // Disable "event is never used" warning.
```
Comment (aleksei-korolev) [inline Tests/Builders/InteractionTrainingBuilderTests.cs:28]: Should we simply remove these events?

Comment (Gusinuhe) [reply Tests/Builders/InteractionTrainingBuilderTests.cs:28]: We can't, I think those are part of the interface.

Comment (aleksei-korolev) [reply Tests/Builders/InteractionTrainingBuilderTests.cs:28]: Not really, these are private classes.

Comment (Gusinuhe) [reply Tests/Builders/InteractionTrainingBuilderTests.cs:28]: Those events are part of `ISnapZoneProperty`

image

Which these classes inherit from:

image
image

**Pull request merged**